bokeh/models/annotations.py

```diff
@@ -1059,6 +1060,10 @@ class Tooltip(Annotation):
     Whether tooltip's arrow should be shown.
     """)
 
+    tooltip_arrow_color = Color(default="#909599", help="""
```

Comment:
This should be `arrow_fill_color` to be consistent with other naming

Reply:
And as long as this is being added, `arrow_fill_alpha` would be nice

Reply:
Thanks for the suggestions. I've made both changes locally but will hold off on pushing until I can resolve the IE11 compatibility issue as well.

Example usage (L51 of gapminder example):
```python
plot.add_tools(
    HoverTool(
        tooltips="@Country",
        arrow_fill_color="black",
        arrow_fill_alpha=0.5,
        point_policy="follow_mouse",
    )
)
```

![image](https://user-images.githubusercontent.com/32231895/100964617-088da880-34f7-11eb-9cb4-b7635fb62006.png)

Reply:
I pushed the requested changes (renaming + add alpha) regardless of the IE11 compatibility fix